docs/tutorials/committing-liquidity.md

```diff
@@ -0,0 +1,143 @@
+
+Your proposed fee factor is used, with other proposed fee factors, to determine the market's liquidity fee. Once the fee for the market is set, all liquidity orders charge that fee, regardless of whether the provider's submitted fee was higher or lower, and whether you propose a higher (or lower) factor, you are still a liquidity provider. The proposed fees are used to calculate the actual fee each participant will pay on a trade in that market.
+
+:::note Read more 
+[Proposing liquidity fee](../concepts/liquidity/rewards-penalties.md#determining-the-liquidity-fee-percentage)
+[Liquidity rewards](../concepts/liquidity/rewards-penalties.md)
+:::
+
+#### List available markets [WIP]
+You'll need to identify which market you want to supply liquidity to, and add the market ID to the submission transaction. 
+
+API LINK
+```bash
+```
+
+## Creating a liquidity commitment
+This tutorial describes how to create, amend or cancel, and send a liquidity commitment submission.
+
+**The liquidity commitment submission must include**:
+* The **market’s unique ID**, denoted as `marketId` - Confirm that the market is in a state to accept liquidity commitment, and is not a rejected market, has not had trading terminated, or has not been settled 
+* **Liquidity commitment amount**: The amount of funds that you want to allocate to providing liquidity. The amount will be moved into a bond account during the duration of your liquidity commitment, denoted as `commitmentAmount`
+* **Proposed liquidity fee**: The scaling factor for the fee you are bidding to receive when your order is matched, on a scale between 0 and 1. For example, a fee level of 0.01 would mean `0.01 * total trade amount` is charged. Note: Your proposed fee is used along with other proposed fees and commitments to determine the actual fee percentage for the market. [Learn how all proposed fee levels influence the market's fees]). Denoted as `fee` Denoted as `fee`
+
+**To submit the liquidity commitment message, you'll also need**: 
+
+* Public key: The public key being used to place the liquidity commitment
+* Propagate: Is true or false. Propagate is used to define if you want the liquidity commitment sent to the nodes for processing immediately (true), or if you want to manually submit the orders in a transaction (false). Note: If you choose to manually submit, it must be within the block tolerance level or it will be rejected
+
+Note: The **commitment amount** is interpreted by the asset decimal place, and expressed in an integer. In the example below, if the asset has a decimal precision of 2, the commitment amount would be 100.
+
+
+```python
+submission = {
+    "liquidityProvisionSubmission": 
+    {
+        "marketId": marketID,
+        "commitmentAmount": "10000",              
+        "fee": "0.01",
+       },
+    "pubKey": pubkey,
+    "propagate": True
+}
+```
+## Tactics for providing liquidity [WIP]
```

Comment:
I would call this "Methods" rather than "Tactics", sounds less "trading advice"-y

Reply:
Yes, good point!